migrator/migrations/1_create_users_table.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE public.users
+(
+    id uuid NOT NULL,
+    username text NOT NULL,
+    "password" text NOT NULL,
+    email text NOT NULL,
+    firstname text NOT NULL,
```

Comment:
If some column has a name containing two words it's a good practice to use "snake_case" to name them.

Reply:
Fix it